### DISCUSSION TITLE
follow up on tuning solver_ref and solver_imp params

### DISCUSSION THREAD

**Discussion by shivanimall** [google-deepmind/mujoco#98] [Asking for Help] [answered]
Hi @yuvaltassa thanks for the response on #63. re-posting so that the q doesn't get missed.

I tried reducing solref for all geoms to [0.004, 1.], plus trying control time-step range from 1/20 to 1/100 but facing the same issue. To note, I use composite objects with different sizes in different episode runs. (I create procedurally generated objects in robosuite, as  [here](https://robosuite.ai/docs/modules/objects.html#creating-a-procedurally-generated-object))

Would I need to further fine-tune based on object size? Many thanks for your help!

**Comment by yuvaltassa** [comment, marked as answer]
Following up in #63.
